Add new examples to example list for CI.

adbridge opened the pull request:
### Description

<!-- 
    Required
    Add here detailed changes summary, testing results, dependencies 
    Good example: https://os.mbed.com/docs/latest/reference/workflow.html (Pull request template)
-->
The following examples have been added:

https://github.com/ARMmbed/mbed-os-example-thread-statistics
https://github.com/ARMmbed/mbed-os-example-sys-info
https://github.com/ARMmbed/mbed-os-example-cpu-usage
https://github.com/ARMmbed/mbed-os-example-cpu-stats

### Pull request type

<!-- 
    Required
    Please add only one X to one of the following types. Do not fill multiple types (split the pull request otherwise).
    Please note this is not a GitHub task list, indenting the boxes or changing the format to add a '.' or '*' in front
    of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
    description heading and to add a 'x' to the correct box.
-->
    [ ] Fix
    [x] Refactor
    [ ] New target
    [ ] Feature
    [ ] Breaking change

adbridge committed:
Add new examples to example list for CI.

The following examples have been added:

https://github.com/ARMmbed/mbed-os-example-thread-statistics
https://github.com/ARMmbed/mbed-os-example-sys-info
https://github.com/ARMmbed/mbed-os-example-cpu-usage
https://github.com/ARMmbed/mbed-os-example-cpu-stats

adbridge commented:
/morph build

cmonr commented:
/morph build

Looking at the CI queue, this might get hit by the Austin IT scheduled downtime.

cmonr commented:
Killed the duplicate build job.

adbridge committed:
Add 2 more examples and remove deprecated one.

Removed:
mbed-os-example-fat-filesystem
Added:
mbed-os-example-error-handling
mbed-os-example-blockdevice

cmonr commented on tools/test/examples/examples.json:
```diff
@@ -318,6 +303,33 @@
+        "mbed": [],
+        "test-repo-source": "github",
+        "features" : [],
+        "targets" : [],
```
K64F?

cmonr reviewed (dismissed):
Nvm, should work for all.

cmonr commented:
/morph build

cmonr commented:
@geky mbed-os-example-blockdevice is not compatible with all devices. Copy editing examples to only run on K64F and restarting CI.

Commit:
Updated blockdevice example to target only K64F

cmonr commented:
/morph build
